libs/datasets/timeseries.py

```diff
@@ -133,8 +128,32 @@ def get_subset(
 
         return self.__class__(data)
 
+    def get_data_for_fips(self, fips) -> List[dict]:
```

Comment:
as a rule of thumb, can we replace `data` with the type of data being returned?

```suggestion
    def get_records_with_fips(self, fips) -> List[dict]:
```